code/modules/clothing/suits/storage.dm

```diff
@@ -157,6 +157,14 @@
 	pockets.max_storage_space = 8
 	pockets.cant_hold |= list(/obj/item/tool_upgrade/armor) //Prevents a bug
 
+/obj/item/clothing/suit/storage/vest/breach2/New()
+	..()
+	pockets = new/obj/item/storage/internal(src)
+	pockets.storage_slots = 5	//five slots
+	pockets.max_w_class = ITEM_SIZE_NORMAL		//fits two normal size items as its big pockets
```

Comment:
Absolutely not. This needs to change down 1 size category or the max storage slots needs to go down. Did you even test this?

![image](https://github.com/user-attachments/assets/716b6340-527a-4152-a83e-2586430281c0)